cockroach-commands.md

```diff
@@ -38,4 +36,18 @@ Flag | Description
 `--logtostderr` |  Write log messages of all severities to the standard error stream and not to log files. If this flag is set to `true`, `--log-dir` and `--alsologtostderr` are ignored. Possible values: `true` or `false`.<br><br>**Default:** `false`
 `--no-color` | Do not colorize the standard error stream based on severity. Possible values: `true` or `false`. <br><br>**Default:** `false`   
 
-The `--log-backtrace-at`, `--verbosity`, and `--vmodule` flags are intended for internal debugging. 
\ No newline at end of file
+The `--log-backtrace-at`, `--verbosity`, and `--vmodule` flags are intended for internal debugging. 
+
+## Environment Variables
+
+For many common `cockroach` flags, such as `--port` and `--user`, you can set environment variables once instead of manually passing the flags each time you execute commands. 
+
+- To find out which flags support environment variables, see the documentation for each [command](#commands). 
+- To output the current configuration of CockroachDB environment variables, run `./cockroach debug env`. 
```

Comment:
We should either leave this command undocumented or move it out of `debug`. I would leave it out.